samples/Sentry.Samples.EntityFramework/Program.cs

```diff
@@ -1,24 +1,76 @@
+using System;
 using System.ComponentModel.DataAnnotations;
 using System.Data.Common;
 using System.Data.Entity;
+using System.Linq;
 using Sentry;
 
-var _ = SentrySdk.Init(o =>
+_ = SentrySdk.Init(o =>
```

Comment:
Is the SDK getting disposed (events getting flushed at the end of the program?)
```suggestion
using var _ = SentrySdk.Init(o =>
```